neural_compressor/torch/quantization/config.py

```diff
@@ -66,6 +67,7 @@ class RTNConfig(BaseConfig):
         "dtype",
         "bits",
         "group_size",
+        # "group_dim", TODO: RuntimeError: group_dim=0
```

Comment:
Please remove the comment once the https://github.com/intel/neural-compressor/pull/1595  is merged.

Reply:
got it.

Reply:
remove in https://github.com/intel/neural-compressor/pull/1561/commits/f7b3981744926704d04bf14fbf3134b7d66b0fea